pkg/capabilities/consensus/ocr3/datafeeds/feeds_aggregator.go

```diff
@@ -112,12 +113,12 @@ func (a *dataFeedsAggregator) Aggregate(lggr logger.Logger, previousOutcome *typ
 		previousReportInfo := currentState.FeedInfo[feedIDStr]
 		feedID, err2 := datastreams.NewFeedID(feedIDStr)
 		if err2 != nil {
-			lggr.Errorw("could not convert %s to feedID", "feedID", feedID)
+			lggr.Errorw("could not convert %s to feedID", logkey.FeedID, feedID)
```

Comment:
Is `logkey` the best package name? We might want to use these with Prometheus too, and so we could generalize and shorten to just `keys`:
```suggestion
			lggr.Errorw("could not convert %s to feedID", keys.FeedID, feedID)
```
But this would have name collisions in some places.

Something like `labels` is perhaps less collision prone, but more verbose:
```suggestion
			lggr.Errorw("could not convert %s to feedID", labels.FeedID, feedID)
```

Any other ideas?

Reply:
logconst or logentry?

Reply:
I think it would need to be fewer than 6 characters to be an improvement on `logkey`

Reply:
Would `package lk` be too opaque? Typically they should be meaningful, but in this case the usage should always make it clear :thinking: It would also make aliasing packages less obnoxious

Reply:
Update to `package lk` to try it out. I think I like it.